core/src/main/java/io/questdb/cairo/mv/MatViewRefreshJob.java

```diff
@@ -584,14 +607,16 @@ private boolean refreshDependentViewsIncremental(
                     try {
                         refreshed = refreshIncremental0(state, baseTableToken, walWriter, refreshTriggeredTimestamp);
                     } catch (Throwable th) {
-                        LOG.error().$("error refreshing materialized view [view=").$(viewToken).$(", error=").$(th).I$();
```

Comment:
is this removed intentionally?

Reply:
yeah, suspected double-logging, we need to review error handling. I chatted on slack